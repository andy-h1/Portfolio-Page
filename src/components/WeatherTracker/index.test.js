import React from "react";
import { render } from "@testing-library/react";
import { WeatherTracker } from ".";

const props = {
  city: "London",
  country: "England",
  image: "01d",
  imgAlt: "sunny",
  timeStamp: 1598656029,
  weatherDesc: "clear sky",
  weatherTemp: 34,
  weatherTempFeelsLike: 36,
  wind: 1.0,
};

describe("Weather Tracker Component", () => {
  it("should render", () => {
    const { container } = render(<WeatherTracker {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
