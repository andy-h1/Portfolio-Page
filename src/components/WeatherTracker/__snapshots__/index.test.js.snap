// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather Tracker Component should render 1`] = `
<div
  class="sc-AxjAm jMqLJl"
  data-testid="weatherTracker"
>
  <img
    alt="sunny"
    class="sc-AxirZ bxhEMV"
    height="auto"
    src="clearDay.png"
    width="250px"
  />
  <table
    class="sc-AxiKw hwcIqX"
  >
    <tbody>
      <tr>
        <th
          class="sc-AxhCb jRGsGb"
        >
          Current Date:
        </th>
         
        <td
          class="sc-AxhUy cwlNFR"
        >
          8/29/2020
        </td>
      </tr>
      <tr>
        <th
          class="sc-AxhCb jRGsGb"
        >
          Time of Weather:
        </th>
        <td
          class="sc-AxhUy cwlNFR"
        >
          12:07:09 AM
        </td>
      </tr>
      <tr>
        <th
          class="sc-AxhCb jRGsGb"
        >
          Location:
        </th>
         
        <td
          class="sc-AxhUy cwlNFR"
        >
          London
          , 
          England
        </td>
      </tr>
      <tr>
        <th
          class="sc-AxhCb jRGsGb"
        >
          Current Temperature:
        </th>
         
        <td
          class="sc-AxhUy cwlNFR"
        >
          34
          °
        </td>
      </tr>
      <tr>
        <th
          class="sc-AxhCb jRGsGb"
        >
          Feels like:
        </th>
        <td
          class="sc-AxhUy cwlNFR"
        >
          36
          °
        </td>
      </tr>
      <tr>
        <th
          class="sc-AxhCb jRGsGb"
        >
          Forecast:
        </th>
         
        <td
          class="sc-AxhUy cwlNFR"
        >
          clear sky
        </td>
      </tr>
      <tr>
        <th
          class="sc-AxhCb jRGsGb"
        >
          Wind speed:
        </th>
         
        <td
          class="sc-AxhUy cwlNFR"
        >
          1
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
